tests/test_children.py

```diff
@@ -0,0 +1,25 @@
+from n2y.notion import Client
+from n2y.notion_mocks import mock_block, mock_rich_text
+from n2y.utils import id_from_share_link
+
+
+def test_create_and_delete_blocks():
+    url = "https://www.notion.so/innolitics/Test-Page-46bc615bed5a479e8390d9373279426f"
```

Comment:
We should use the same notion pages and access token we use for the end to end tests here. See the doc string at the top of tests/end_to_end_tests.py